indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -56,3 +67,55 @@ def _checkParams(self, params):
 
     def _process(self):
         return self.abstract_file.send()
+
+
+class RHDisplayReviewableTracks(RHAbstractsBase):
+    def _checkProtection(self):
+        RHAbstractsBase._checkProtection(self)
+        if not session.user:
+            raise Forbidden
+
+    def _process(self):
+        query = Track.query.with_parent(self.event_new)
+        track_reviewer_abstract_count = get_track_reviewer_abstract_counts(self.event_new, session.user)
+
+        # if the user is not a global convener only show their tracks
+        if (session.user not in self.event_new.global_abstract_reviewers and
+                session.user not in self.event_new.global_conveners):
```

Comment:
Probably because it's much clearer that way where the body starts:

```python
if (foo or
    bar):
    foobar()

if (foo or
        bar):
    foobar()
```